scio-bigquery/src/it/scala/com/spotify/scio/bigquery/types/StorageIT.scala

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2019 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.bigquery.types
+
+import com.google.protobuf.ByteString
+import com.spotify.scio._
+import com.spotify.scio.bigquery._
+import org.apache.beam.sdk.testing.PAssert
+import org.joda.time.{DateTimeZone, Duration, Instant}
+import org.scalatest._
+
+import scala.collection.JavaConverters._
+
+// Run BigQueryITUtil to re-populate tables for integration tests
+class StorageIT extends FlatSpec with Matchers {
```

Comment:
Also any reason not to use the `PipelineSpec`? 😄

Reply:
Because this module doesn't depend on `scio-test`. Or we can move `StorageIT` to `scio-test` which is more consistent with the other ones.